src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java

```diff
@@ -189,17 +210,22 @@ private String getState(CaseData updatedCaseData) {
 
     private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
+        String claimNumber = caseData.getLegacyCaseReference();
 
-        if (caseData.getRespondent1Represented() == YesOrNo.NO) {
+        if (caseData.getRespondent1Represented() == NO) {
             return SubmittedCallbackResponse.builder()
                 .confirmationHeader("# Your claim will now progress offline")
                 .confirmationBody(LIP_CONFIRMATION_BODY)
                 .build();
+        } else if (caseData.getRespondent1OrgRegistered() == NO) {
+            return SubmittedCallbackResponse.builder()
```

Comment:
Should we not clear the reference and organisation fields here in case the user provided a value then changed their mind and selected NO.

Reply:
this is for submitted callback, we don't update data here.